### ISSUE TITLE
Can't open an utterance  [w/219 - est. 3 hours]

### ISSUE THREAD

**Issue by bugspotter33** [DCS-LCSR/SignStream3#220]
I just started non-manuals for Rachel in file 82 of the Rachel/Jonathan dialog. I have only finished U1 and am starting U2 however I can't get to U2 to enter data. Every time I click on U2 on the left, it highlights blue for a second and then goes back to U1. I can show you what I see now but the highlight transition on the screen happens to fast for me to capture. I also can't scroll to U2 from within the open data field window. I've also tried extending the window which allows me to see a portion of the next utterance however I still can't scroll over with the mouse or manually to get to U2.

![screen shot 2018-05-25 at 2 32 30 pm](https://user-images.githubusercontent.com/38887034/40560647-9c85e59e-6028-11e8-8fa8-4853753365dd.png)

![screen shot 2018-05-25 at 2 32 40 pm](https://user-images.githubusercontent.com/38887034/40560650-9f8b99d2-6028-11e8-8a20-d7f3629dd924.png)

**Comment by cneidle** [comment]
I have not been able to reproduce this, nor has Aimee.

**Comment by gregorydimitriadis** [comment]
@bugspotter33 @cneidle testing on the same machine/environment/etc?

**Comment by bugspotter33** [comment]
~~Hi Greg, I have been working on the same machine and environment and I just tried to reproduce the issue and the closest I've gotten is that as I click on the utterances starting from U1 of the Rachel data of file 82, I can see the data for U1 and U2 however when I click on U3 I get of flash of the data and then I am shown data for U4. However, this issue seems similar to issue to the "Redirecting Utterances" bug that I reported last week.~~

**Comment by cneidle** [comment]
MY APOLOGIES!   My comment was posted for the wrong bug !!!
The bug I had been unable to reproduce was something different that had been reported to ne by Aimee !!!  Not this one.  Very sorry.  
Please disregard my comment above about attempting to reproduce this problem.

**Comment by gregorydimitriadis** [comment]
Gotcha - thanks. Indeed, seems similar to #219

**Comment by bugspotter33** [comment]
Hi Greg, I was able to reproduce this bug today as Carol and I were discussing various utterances. It occurred as I was trying to look at a specific utterance (Rachel File 82). I was trying to look at U3 and it kept skipping it and showing me U4. I found that if I used the buttons at the bottom (see attached screenshot) I was able to get to the data of U3.

![screen shot 2018-05-30 at 2 26 43 pm](https://user-images.githubusercontent.com/38887034/40740054-c9597aa4-6415-11e8-8b8c-ce4b8695b1e9.png)
